conn.go

```diff
@@ -108,6 +108,32 @@ func (c *Conn) Write(b []byte) (n int, err error) {
 	return n, nil
 }
 
+// Write writes data to the connection and specific stream
+func (c *Conn) WriteToStream(b []byte, streamID uint16) (n int, err error) {
+	if c.state != StateAspActive {
+		return 0, ErrNotEstablished
+	}
+	d, err := messages.NewData(
+		c.cfg.NetworkAppearance, c.cfg.RoutingContexts, params.NewProtocolData(
+			c.cfg.OriginatingPointCode, c.cfg.DestinationPointCode,
+			c.cfg.ServiceIndicator, c.cfg.NetworkIndicator,
+			c.cfg.MessagePriority, c.cfg.SignalingLinkSelection, b,
+		), c.cfg.CorrelationID,
+	).MarshalBinary()
+	if err != nil {
+		return 0, err
+	}
+
+	c.sctpInfo.Stream = streamID
+	n, err = c.sctpConn.SCTPWrite(d, c.sctpInfo)
+	if err != nil {
+		return 0, err
+	}
```

Comment:
It updates the stream ID set in *Conn, and thus the background messages exchanges will be done with the last set stream ID. Since it seems `func (c *SCTPConn) SCTPWrite(b []byte, info *SndRcvInfo) (int, error)` doesn't update the value after sending a message, I think it is safe to use copied SndRcvInfo.

```suggestion
	info := c.sctpInfo
	info.Stream = streamID
	n, err = c.sctpConn.SCTPWrite(d, info)
	if err != nil {
		return 0, err
	}
```